File: 0001-two-sum/0001-two-sum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        i=0
        j=0
        while i < len(nums) and j < len(nums):
            if i!=j and nums[i] + nums[j] == target:
                return [i,j]
            elif j == len(nums)-1:
                i+=1
                j = i+1
            else:
                j+=1

File: 0001-two-sum/0001-two-sum.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        int i, j;
        int[] a = new int[2];
        int n = nums.length;
        for (i = 0; i < n - 1; i++) {
            for (j = i + 1; j < n; j++) {
                if (nums[i] + nums[j] == target) {
                    a[0] = i;
                    a[1] = j;
                }
            }
        }
        return a;
    }
}